### PR TITLE
Article background image refactor and alt text

### PR DESCRIPTION
**Pull request by jakecatron**

### CODE REVIEW

File: _layouts/article.html [hunk 1]
```diff
@@ -6,15 +6,17 @@
 {% assign author = page.author | get_doc %}
 {% assign topic = page.category | get_doc %}
 {% assign tags = page.tags | get_doc %}
+{% assign alt = page.assets | alt %}
```
Comment (tcmacdonald) [inline _layouts/article.html:9]: Is this necessary? I suspect this isn't actually returning anything.

File: _assets/stylesheets/pages/_articles.scss [hunk 2]
```diff
@@ -20,6 +19,16 @@
         }
       }
     }
+    img {
+      height: 790px;
+      width: 100vw;
+    }
```
Comment (tcmacdonald) [inline _assets/stylesheets/pages/_articles.scss:25]: What if I upload an image to the CMS that is less than 790px tall?

File: _layouts/article.html (hunk 1 above)
Comment (jakecatron) [reply _layouts/article.html]: Yes, you are correct. Just removed that!

**Comment by beckysingson** [comment]
@jakecatron @tcmacdonald Leaving comments here since this work doesn't have a corresponding Rally story at the moment:

The new jumbotron image doesn't respond appropriately on mobile. 

Here's how the deploy preview appears on Safari on iOS:
IMG_4559

Compared to how the article jumbotrons appear currently on INT:
IMG_4558

**Review by kellgauthier — APPROVED** [review]

**Pull request merged**